docs/en/user_guides/1_config.md

```diff
@@ -226,12 +226,44 @@ load_from = None  # Load checkpoint from file.
 resume = False  # Whether to resume from existed model.
 ```
 
+These are all the configs for training and testing PSPNet, we use [MMEngine](https://github.com/open-mmlab/mmengine) to load and parse these configs.
```

Comment:
we use [MMEngine](https://github.com/open-mmlab/mmengine) to load and parse these configs.- > 
```
To load and parse them, we can use [`Config`](URL cfg tutorial in English) implemented in [MMEngine](https://github.com/open-mmlab/mmengine)
```